RevenueCatUI/CustomerCenter/Views/RestorePurchasesAlert.swift

```diff
@@ -164,7 +162,7 @@ struct RestorePurchasesAlert: ViewModifier {
 
     private func dismissAlert() {
         self.alertType = .restorePurchases
-        dismiss()
+        self.isPresented = false
```

Comment:
also found this when testing - if we call `dismiss` from SwiftUI, we'll dismiss the entire thing instead of just the alert. Feels like we just want to dismiss the alert, right?

Reply:
Oh interesting 🤔 

Yeah, we want to dismiss only the alert. It looks like a bug. I will test and probably open a separate PR with this only